.storybook/themes/commerce.scss

```diff
@@ -881,8 +881,8 @@
 					grid-template-rows: 1fr,
 					gap: var(--global-spacing-5),
 					margin: auto,
-					max-width: calc(var(--content-max-width) * 1px),
-					width: var(--content-scale-width),
+					max-width: 100%,
+					width: 100%,
```

Comment:
`var(--content-scale-width)` will need to be applied to each of the component styles with a `margin: auto`